decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -52,6 +54,14 @@ def permissions
 
       private
 
+      def view_private_users_menu?
```

Comment:
Related to the renaming before 
```suggestion
      def user_can_read_private_users?
```

(If you accept these you should implement this in participatory spaces permissions too)

Reply:
fixed in [bb74aa7](https://github.com/decidim/decidim/pull/12031/commits/bb74aa7c35909f58337992441e4040338a5c8537)